.cypress/integration/services.spec.js

```diff
@@ -53,7 +61,16 @@ describe('Testing services table', () => {
 
 describe('Testing service view empty state', () => {
   beforeEach(() => {
-    cy.visit(`app/opendistro-trace-analytics#/services/${SERVICE_NAME}`);
+    // exception is thrown on loading EuiDataGrid in cypress only, ignore for now
```

Comment:
Just want to verify based on this comment, this does not break any workflows right?

Reply:
should not affect anything, seems to be a conflict with the data grid component and some nested flex items inside cypress. I couldn't see the error when testing manually